### DISCUSSION TITLE
客户端报错无法启动

### DISCUSSION THREAD

**Discussion by StephenSSR** [zdz/ServerStatus-Rust#46] [Q&A] [answered]
在debian10上，服务端是最新的1.50，遇到如下报错
`sys id: cf2b4c4d69c50d5b94fd62361f154832
sys info: {"name":"h2","version":"1.5.0","os_name":"linux","os_arch":"x86_64","os_family":"unix","os_release":"Linux 10 Debian GNU/Linux","kernel_version":"5.15.41-bbrplus","cpu_num":2,"cpu_brand":"Intel(R) Xeon(R) CPU E5-2697 v2 @ 2.70GHz","cpu_vender_id":"GenuineIntel","host_name":"racknerd-10e9e5"}
enable feature native
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Uncategorized, error: "failed to lookup address information: Name does not resolve" }', client/src/status.rs:331:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Aborted`
请问是什么原因

**Comment by zdz** [comment, marked as answer]
看样子是 dns 解析  `xxx.tz.cloudcpp.com` ping 的那三个域名失败了，检查一下那三个 域名的解析情况
或者使用 `--disable-ping` 禁止 ping 丢包探测，
或者 参照这里 https://doc.ssr.rs/faq/#ping 自定义 ping 的地址
还有，你这个应该是客户端吧

**Comment by StephenSSR** [comment]
是的，打错了，是客户端，也确实是dns的问题，cm和cu没有ping通
